internal/node/node_launcher.sh

```diff
@@ -129,8 +129,19 @@ for ARG in "${ALL_ARGS[@]}"; do
   esac
 done
 
+# Note: Bash does not forward any termination signals to any child process when running in docker
```

Comment:
nit: can you wrap this at the same length as the other comments? (looks like 80 columns)